packages/appstore/src/components/wallet-content/wallet-content.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import classNames from 'classnames';
+import ContentDivider from './wallet-content-divider';
+import { isMobile } from '@deriv/shared';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { TCoreStores } from '@deriv/stores/types';
+import WalletCfdsListing from './wallet-cfds-listing';
+import WalletOptionsAndMultipliersListing from './wallet-option-multipliers-listing';
+import './wallet-content.scss';
+
+type TProps = {
+    is_demo: boolean;
+    is_eu: boolean;
+    account: TCoreStores['client']['accounts'][0];
```

Comment:
name it wallet_account for consistency
```suggestion
    wallet_account: TCoreStores['client']['accounts'][0];
```